cmd/cli/install_test.go

```diff
@@ -1064,6 +1080,7 @@ var _ = Describe("Resolving values for install command with cert-manager paramet
 				"enablePrometheus":               true,
 				"enableGrafana":                  true,
 				"deployJaeger":                   false,
+				"envoyLogLevel":                  testEnvoyLogLevel,
```

Comment:
In addition to the changes, could you add a test to toggle the install option for the log level and verify they resolve correctly to chart values. Ex. https://github.com/openservicemesh/osm/blob/main/cmd/cli/install_test.go#L1138

Reply:
added a new test

Reply:
added a new test